cdc_rsync/server_arch.h

```diff
@@ -33,6 +33,12 @@ class ServerArch {
   // starting with C: indicate Windows.
   static Type Detect(const std::string& destination);
 
+  // Returns the arch type that matches the current process's type.
```

Comment:
Nit: `arch`-> `architecture`

Reply:
Went the other way, calling all architectures arches now.